pkg/metrics/metrics.go

```diff
@@ -0,0 +1,90 @@
+package metrics
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	// Default values which are to be used as labels
+	// for prometheus metrics.
+	ERROR      = "opsrc_error"
+	OPSRC_NAME = "opsrc_name"
+
+	// Path, Port and Host where custom metrics would be exposed.
+	METRICSPATH = "/metrics"
+	METRICSPORT = "8383"
+	METRICSHOST = "0.0.0.0"
+
+	// Error message to be returned when quay.io is down.
+	UNREACHABLE_ERROR = "Quay.io is unavailabe"
```

Comment:
it's a lot easier to query prometheus when label values don't have spaces. What about something like `quay_unavailable`, `no_host`, etc?